api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Thinking about this...

Users and contributors are different things.  If I just navigate to a user detail endpoint `/v2/users/me`, the unregistered_contributor field doesn't apply here.  Unregistered contributor names only come into play when I'm asking for contributors on a node.  You might have called the user Robert, I called him Bob, and until the user claims the account, his name is different on different nodes.  But the different names are associated with the nodes.